### PR TITLE
[jit] add 'abs' builtin

### PR DESCRIPTION
**Pull request by TheCodez**

### CODE REVIEW

File: torch/csrc/jit/register_prim_ops.cpp [hunk 1]
```diff
@@ -1748,6 +1748,23 @@ RegisterOperators reg2({
...
+          push(stack, x < 0 ? -x : x);
+          return 0;
+        }),
+    Operator(
+        "prim::abs(float x) -> float",
+        [](Stack& stack) {
+          float x;
+          pop(stack, x);
+          push(stack, x < 0 ? -x : x);
+          return 0;
```
Comment (eellison) [inline torch/csrc/jit/register_prim_ops.cpp]: Could you use std::abs instead ? might be more optimized...

File: torch/csrc/jit/register_prim_ops.cpp [hunk 2]
```diff
@@ -1748,6 +1748,23 @@ RegisterOperators reg2({
     DEFINE_INT_OP(aten::__or__, a | b),
     DEFINE_INT_OP(aten::__xor__, a ^ b),
 
+    Operator(
```
Comment (eellison) [inline torch/csrc/jit/register_prim_ops.cpp]: Could you also add an op for `prim::abs(Tensor x)` ?

**Review by eellison — COMMENTED** [review]
Looks good! Couple small comments

File: torch/csrc/jit/register_prim_ops.cpp [hunk 3]
```diff
@@ -1748,6 +1749,31 @@ RegisterOperators reg2({
...
+        "prim::abs(int x) -> int",
+        [](Stack& stack) {
+          int64_t x;
+          pop(stack, x);
+          push(stack, static_cast<int64_t>(std::abs(x)));
```
Comment (eellison) [inline torch/csrc/jit/register_prim_ops.cpp]: Sorry, I didn't realize std::abs wasn't defined for ints as well. Could you put it back to what you had previously?

**Review by eellison — APPROVED** [review]
Looks good! thanks for doing this. one small comment correcting what i said earlier 🤪

File: torch/csrc/jit/register_prim_ops.cpp (hunk 3 above)
Comment (TheCodez) [reply torch/csrc/jit/register_prim_ops.cpp]: It's defined for ints as well, the cast was just plain unnecessary.

**Pull request closed without merging**